interfaces/webusb.idl

```diff
@@ -1,5 +1,6 @@
-// Copied from https://wicg.github.io/webusb/#idl-index minus the
```

Comment:
@reillyeon, you wrote this bit. What do you think about our auto-updating scheme here? The resulting tests will fail, is that OK? Should it be removed from the spec until it's ready to test and implement, perhaps?

Reply:
I think the spec should be annotated to mark that section as unstable and if we have an automatic update mechanism it should be possible to mark IDL sections as "unstable and untested" so they aren't included.

Reply:
That would work for us I think. @tabatkins, you have some output from Bikeshed that's used to scrape definitions and things, what you suggest as the "API" for a spec saying that pieces are not to be implemented/tested yet?

Reply:
@tabatkins - Ping. Is there an annotation for ignoring an IDL chunk?

Reply:
Yes, if it's in a non-normative section (`.note`, `.example`, `.non-normative` or `.informative` on an ancestor) it gets ignored for definition purposes in Bikeshed, and if it's non-normative or has `.extract` on the `pre` (I dunno where this practice came from but it's common on the WHATWG side) it gets omitted from the IDL index at the end of the spec.

Reply:
The reason these definitions are there is partially because of how the Permissions API defines the permissions storage data model. What would the best way of defining IDL for objects that are only included in internal slots?